key_datarefs.md

```diff
@@ -0,0 +1,101 @@
+# Identifying Key DataRefs for Pilot Performance
+
+##### Preliminary Notes
+1. Automatic type conversion is NOT done for you. You need to know the exact data types of the data you reference.
+2. Data types are found as a SET which means you can choose which one you want to use, i.e. double or float.
+3. Detailed DataRef information can be found at the following links:
+    - "https://www.siminnovations.com/xplane/dataref/?name=&type=&writable=&units=&description=&submit=Search"
+    - "https://developer.x-plane.com/datarefs/"
+4. The values listed may not be the ones we actually want for our plugin, as these all correspond to the true flightmodel readings. There is an argument that DataRefs from gauges and instruments (which may be faulty in the simulator purposefully/accidentally) should be read instead to provide the data the pilot using to fly. I plan to discuss this with my group on Monday during our code reviews and confirm.
+5. I am unsure what the github issue meant by finding the usage of these DataRefs. Am I to figure out what they are used for within XPlane itself? Or do I need to figure out what we will be using them for in our plugin? I plan on also raising this issue with my group during our Monday capstone meeting.
```

Comment:
When you are not sure about something on the issue, you should post the question in the issue. Don't post your questions in the documentation you are writing.